iris_grib/_save_rules.py

```diff
@@ -731,13 +733,34 @@ def set_forecast_time(cube, grib):
     gribapi.grib_set(grib, "forecastTime", fp)
 
 
-def set_fixed_surfaces(cube, grib):
+def set_fixed_surfaces(cube, grib, full3d_cube=None):
 
     # Look for something we can export
     v_coord = grib_v_code = output_unit = None
 
+    # Detect factories for hybrid vertical coordinates.
+    hybrid_height_factories = [
+        factory for factory in cube.aux_factories
+        if isinstance(factory, HybridHeightFactory)]
+    if not hybrid_height_factories:
+        hybrid_height_factory = None
+    else:
+        # If any, there should be just one.
+        factory, = hybrid_height_factories
+        # Fetch the matching 'complete' factory from the *full* 3d cube, so we
+        # have all the level information.
+        hybrid_height_factory = full3d_cube.aux_factory(factory.name())
+
+    # hybrid pressure
```

Comment:
This comment says 'hybrid pressure', but it's looking for a hybrid height factory with a vertical coordinate code 118, which is specifically hybrid height, not hybrid pressure.  Is it just the comment that's misplaced?

Reply:
>  Is it just the comment that's misplaced?

Yes I think so.
Good spot :trophy: